examples/ecs/event.rs

```diff
@@ -12,6 +12,8 @@ fn main() {
         .run();
 }
 
+#[derive(Event)]
+#[store(3)]
```

Comment:
This needs a comment explaining what it does and why you might want to use it ;)